pwiz_tools/Skyline/Controls/IonTypeSelectorControl.cs

```diff
@@ -339,9 +339,12 @@ public void Update(GraphSpectrumSettings set, PeptideSettings peptideSet)
                     //Add the buttons that are not there yet
                     if (!Controls.OfType<CheckBox>().Any(cb => loss.Equals(cb.Tag)))
                     {
+                        string lossText = string.Format(CultureInfo.CurrentCulture, @"-{0:F0}", loss.AverageMass);
+                        if (loss.Charge > 0)
+                            lossText += Transition.GetChargeIndicator(loss.Charge);

```

Comment:
I feel this might be confusing for the user to have something like -12.124- Maybe some kind of brackets might be useful: -[12.124-]